ironfish/src/primitives/mintData.ts

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export interface MintData {
+  assetId?: Buffer
+  name: string
+  metadata: string
+  value: bigint
+  isNewAsset: boolean
```

Comment:
We should delete this, this isnt a concept in ironfish. We don't differentiate between new mints and old mints

Reply:
this is for serialization because asset id is optional. I need to know if there is asset id in deserialization.